integration/helper_test.go

```diff
@@ -2,22 +2,46 @@ package integration
 
 import (
 	"context"
+	"errors"
+	"os"
 
 	"github.com/TykTechnologies/tyk-operator/api/model"
 	"github.com/TykTechnologies/tyk-operator/api/v1alpha1"
+	"github.com/TykTechnologies/tyk-operator/pkg/environmet"
+
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	cr "sigs.k8s.io/controller-runtime/pkg/client"
+	e2eKlient "sigs.k8s.io/e2e-framework/klient"
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 )
 
-func createTestAPIDef(ctx context.Context, namespace string, mutateFn func(*v1alpha1.ApiDefinition),
-	envConf *envconf.Config,
-) (*v1alpha1.ApiDefinition, error) {
-	client := envConf.Client()
+const (
+	testSubGraphCRMetaName = "test-subgraph"
+	testSubGraphSchema     = "test-schema"
+	testSubGraphSDL        = "test-SDL"
+)
+
+// createTestClient creates controller-runtime client by wrapping given e2e test client. It can be used to create
+// Reconciler for objects such as ApiDefinition
+func createTestClient(k e2eKlient.Client) (cr.Client, error) {
```

Comment:
I really loved this! We won't need to have a running Operator instance for integration tests!